Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/StubIrBridgeBuilder.kt

```diff
@@ -16,6 +19,44 @@ class BridgeBuilderResult(
         val excludedStubs: Set<StubIrElement>
 )
 
+private data class CCalleeWrapper(val name: String, val lines: List<String>)
+
+/**
+ * Some functions doesn't have an address (e.g. macros-based or builtins).
+ * To solve this problem we generate a wrapper function.
+ */
+private fun generateCCalleeWrapper(cCallSymbolName: String, origin: StubOrigin.Function): CCalleeWrapper =
+        if (origin.function.isVararg) {
+            CCalleeWrapper(origin.function.name, emptyList())
+        } else {
+            val function = origin.function
+            val wrapperName = "${cCallSymbolName}_wrapper"
+            val returnType = function.returnType.getStringRepresentation()
+            val parameters = function.parameters.mapIndexed { index, parameter ->
+                val parameterName = parameter.name
+                val name = if (parameterName == null || parameterName.isEmpty()) {
+                    "p$index"
+                } else {
+                    parameterName
+                }
+                name to parameter.type.getStringRepresentation()
+            }
+
+            val callExpression = "${function.name}(${parameters.joinToString { it.first }});"
+            val wrapperBody = if (function.returnType is VoidType) {
```

Comment:
Does this work for functions with return type being typedef to void?
We definitely need a test for this.